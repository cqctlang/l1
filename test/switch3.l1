
x = 1;
switch(x){
case 1:
case 4:
	printf("case 1 or 4\n");
	break;
case 2:
	printf("case 2\n");
	break;
default:
	printf("default\n");
	break;
}

x = 2;
switch(x){
case 1:
case 4:
	printf("case 1 or 4\n");
	break;
case 2:
	printf("case 2\n");
	break;
default:
	printf("default\n");
	break;
}

x = 1;
switch(x){
case 1:
case 4:
{
	printf("case 1 or 4\n");
	break;
}
case 2:
{
	printf("case 2\n");
	break;
}
default:
{
	printf("default\n");
	break;
}
}

x = 2;
switch(x){
{
case 1:
case 4:
	printf("case 1 or 4\n");
	break;
case 2:
	printf("case 2\n");
	break;
}
default:
	printf("default\n");
	break;

}
