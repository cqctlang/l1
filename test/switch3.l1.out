case 1 or 4
case 2
case 1 or 4
case 2
